### PR TITLE
Feature/geometry data timeline

### PR DESCRIPTION
**Pull request by alegnaaived**
The file src/api/tmp_hf_timeline.json was forced to have key and id. To implement NIVO component "Line" is required "id", and Search component uses "key".

### CODE REVIEW

File: src/charts/MultiLinesGraph.jsx [hunk 1]
```diff
@@ -59,16 +59,16 @@ class MultiLinesGraph extends React.Component {
...
     });
     this.setState({ data: transformedData, selectedId });
   };
 
   selectLine = (point) => {
     const { onClickGraphHandler } = this.props;
-    this.changeSelected(point.serieId || point.id);
-    onClickGraphHandler(point.serieId || point.id);
+    this.changeSelected(point.serieId || point.id || point.key);
+    onClickGraphHandler(point.serieId || point.id || point.key);
```
Comment (erikasv) [inline src/charts/MultiLinesGraph.jsx:71]: We can't just keep increasing this list size. See general comment and when we decide what to do, make the necessary change

Comment (alegnaaived) [reply src/charts/MultiLinesGraph.jsx:71]: The issue is that: key has been used as the preferred standard but id is used in NIVO Line component as default. That is why I used both https://nivo.rocks/line/

Maybe we could keep key as the unique and standardized field in the response from the API and create a function before sent the data to Line component.

Comment (erikasv) [reply src/charts/MultiLinesGraph.jsx:71]: That's one of the options I present in the general comment

File: src/api/tmp_hf_timeline.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [
   {
     "id": "paramo",
+    "key": "paramo",
```
Comment (erikasv) [inline src/api/tmp_hf_timeline.json:4]: According to what we discussed, remove the id attribute

File: src/charts/MultiLinesGraph.jsx (hunk 1 above)
Comment (alegnaaived) [reply src/charts/MultiLinesGraph.jsx]: Opt1 Implemented

File: src/api/tmp_hf_timeline.json (hunk 2 above)
Comment (alegnaaived) [reply src/api/tmp_hf_timeline.json:3]: Done

File: src/charts/MultiLinesGraph.jsx [hunk 3]
```diff
@@ -14,10 +14,20 @@ class MultiLinesGraph extends React.Component {
...
+    this.setData(data, colors);
+  }
+
+  /**
+   * Add fields required in data by other components, including NIVO Line (field id)
+   *
+   * @param {Object} data API response to be rendered
+   * @param {Func} colors handle color to be asigned by data type
+   */
+  setData = (data, colors) => {
```
Comment (erikasv) [inline src/charts/MultiLinesGraph.jsx:26]: I don't see the necessity of this function, componentDidMount is doing nothing else, this adds complex to the component needlessly

File: src/charts/MultiLinesGraph.jsx [hunk 4]
```diff
@@ -59,8 +69,8 @@ class MultiLinesGraph extends React.Component {
   changeSelected = (selectedId) => {
```
Comment (erikasv) [inline src/charts/MultiLinesGraph.jsx:69]: Please add documentation to this function

**Pull request merged**